reports/executive_fullfilment_requests/entrypoint.py

```diff
@@ -0,0 +1,543 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2021, CloudBlue
+# All rights reserved.
+#
+import pathlib
+from datetime import date
+from tempfile import NamedTemporaryFile
+from collections import namedtuple
+from math import ceil
+from copy import deepcopy
```

Comment:
Please sort these imports.

Reply:
Thanks! Added flake8 that will check it!